src/Application.vala

```diff
@@ -31,22 +31,22 @@ public class Music.Application : Gtk.Application {
             default_width = 650,
             title = _("Music")
         };
-        main_window.show_all ();
+        main_window.present ();
 
-        Gtk.IconTheme.get_default ().add_resource_path ("/io/elementary/music");
+        Gtk.IconTheme.get_for_display (Gdk.Display.get_default ()).add_resource_path ("/io/elementary/music");
```

Comment:
Alternatively, setting `resource-base-path` on the app before startup should do the same thing for you

Reply:
Thanks for the tip!!